### ISSUE TITLE
Adding support for ImplicitCAD's modified OpenSCAD syntax

### ISSUE THREAD

**Issue by ronasuti** [farrellm/scad-clj#38]
Considering ImplicitCAD is made to be semi-interoperable with OpenSCAD's syntax, I was wondering if this software package could add ImplicitCAD support, or if an ImplicitCAD port could be forked from this.

**Comment by julialongtin** [comment]
ImplicitCAD maintainer here. I'd like to see this as well. The biggest problem I see in the one example of output given to me is use of color (which I can implement as a no-op, so no big deal), and the use of hull(). ImplicitCAD's SDF based representation is incapable of representing a hull.

**Comment by julialongtin** [comment]
please see https://github.com/colah/ImplicitCAD/issues/384 for a user wanting to use this with ImplicitCAD.

**Comment by farrellm** [comment]
What's the ask here? Are you looking for `hull` to be translated to some other functions? Or removed because ImplicitCAD doesn't support it?

**Comment by julialongtin** [comment]
either-or, works for me.

**Comment by farrellm** [comment]
Neither of these makes sense to me. scad-clj is a thin wrapper, it doesn't implement any hardcore constructive geometry like `hull`, and removing support for an OpenScad function that people are clearly using is silly. 

Anyone is welcome to write a new module (like `scad-clj.model`) that only exports ImplicitCAD functions, but I have not used ImplicitCAD and am not familiar with the differences.

**Comment by julialongtin** [comment]
fair enough. i haven't looked at the user's code close enough to know how thick the wrapper is. I'll try filing this with the project consuming your library.

**Comment by farrellm** [comment]
I’m closing this, please reopen if there is a specific ask.
